interfaces/portal/src/app/components/page-layout-payment/page-layout-payment.component.ts

```diff
@@ -250,6 +250,39 @@ export class PageLayoutPaymentComponent {
     }),
   );
 
+  readonly isPaymentApproved = computed(() => {
+    if (!this.transactions.isSuccess()) {
+      return false;
+    }
+
+    return this.transactions
+      .data()
+      .some(
+        (transaction) => transaction.status !== TransactionStatusEnum.created,
+      );
+  });
+  readonly statusBadgeLabel = computed(() => {
+    if (!this.transactions.isSuccess()) {
+      return '';
+    }
+
+    if (this.isPaymentApproved()) {
+      return 'Approved';
```

Comment:
this should be lokalized? I'm also doubting if
- this should be based on TransactionStatusEnum.approved and .pendingApproval (these are atm in another PR still), as the strings should match with that. Then again, in the current use they are not transaction states, but payment states.
- Or - by that logic - create a new payment state enum? 
- Then again, we're not using that anywhere else for now, so maybe it's fine. But at least lokalize it.